src/pages/Admin/Admin.tsx

```diff
@@ -264,21 +260,9 @@ const VenueInfoComponent: React.FC<VenueInfoComponentProps> = ({
     );
   }, [venue]);
 
-  const [showDeleteModal, setShowDeleteModal] = useState(false);
+  const { isShown, show, hide } = useShowHide();
```

Comment:
Rename the state variables here so they maintain their semantic context in relation to the delete modal:

```suggestion
  const { isShown: isDeleteModalVisible, show: showDeleteModal, hide: hideDeleteModal } = useShowHide();
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1380/commits/7105aeff532bea24d5bb7b8ca47f2fb586e955d7